addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -10,147 +10,93 @@ Currently, the binding supports a single type of Thing, being the `command` Thin
 
 The binding does not require any specific configuration.
```

Comment:
Please note that careful though went into rewriting parts of the README by the community, also a lot of small markdown errors where fixed looking at changes it seems you might have overwritten some of these changes.

Maybe @Josar can help you out here?

Reply:
@martinvw @Josar I was just force pushing some mods based on the previous feedback round, and I need to further test things before it is completed. I also saw the Linux related changes in the main branch wrt readme.md, so I need to pick those up and re-integrate these in the doc

Reply:
@kgoderis what is the status of your note above:

> @martinvw @Josar I was just force pushing some mods based on the previous feedback round, and I need to further test things before it is completed. I also saw the Linux related changes in the main branch wrt readme.md, so I need to pick those up and re-integrate these in the doc as I do not want to loose the community's inputs